R/scm_support.R

```diff
@@ -44,38 +43,50 @@ equilibrium_quiet <- function(base=Control()) {
   base
 }
 
+##' Hopefully sensible set of parameters for use with the SCM.  Turns
+##' accuracy down a bunch, makes it noisy, sets up the
+##' hyperparameterisation that we most often use.
+##' @title Sensible, fast (ish) SCM parameters
+##' @author Rich FitzJohn
+##' @param type Name of model (defaults to FF16 but any strategy name is valid).
+##' @export
+scm_base_control <- function(type="FF16", env=environment_type(type)) {
+  ctrl <- equilibrium_verbose(fast_control())
+  ctrl$schedule_eps <- 0.005
+  ctrl$equilibrium_eps <- 1e-3
+  return(ctrl)
+}
+
+
+##' This used to be bundled into base control but refactoring to separate
+##' concerns. May disappear in the future.
+scm_base_parameters <- function(type="FF16", env=environment_type(type)) {
```

Comment:
I wonder if we should remove the default type, to force that type needs to be specified?